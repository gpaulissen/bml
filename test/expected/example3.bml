#+TITLE: 

#+DESCRIPTION: 

#VUL 00

#SEAT 0

1C
    1D
        1H = Minimum with 2--3!h
            1S = 4+!h, 4!s, at most invitational
            1N = Sign off
            2C = Puppet to 2!d
                2D = Forced
                    2H = Mildly invitational with 5!h
                    2S = Invitational, 5+!h and 4!s
                    2N = Strongly invitational with 5!h
                    3C = Invitational with 4!h and 5+!c
                    3D = Invitational with 4!h and 5+!d
                    3H = 6!h, about 11--12 hcp
            2D = Artificial game force
            2H = 6+!h, about 9--10 hcp
        1S = 5+!c, 4+!s, unlimited
        1N = 17--19 bal, 2--3!h
        2C = 5+!c, unbal, 0--2!h, 0--3!s
        2D = Reverse
        2H = Minimum, 4!h
        2S = 16+ hcp, 5+!c and 4+!h
            3D = Retransfer
                3H
                    3S = Cue bid, slam interest
                    4C = Cue bid, slam interest
                    4D = Cue bid, slam interest
                    4H = To play
            3H = Invitational
            3S = Splinter
            4C = Splinter
            4D = Splinter
            4H = To play
        2N = 16+ hcp, 6+!c. 18+ if 3!h
            3C = Suggestion to play
            3D = Relay
                3H = 3!h, 18+ hcp
            3H = Game forcing with 6+!h
        3C = 15--17 hcp, 6+!c and 3!h
            3D = Retransfer
            3H = Invitational
        3D = 17--19 bal, 4!h
            3H = To play
        3H = 13--15 hcp, good hand, 5+!c and 4!h
            3N = Asking for singleton
