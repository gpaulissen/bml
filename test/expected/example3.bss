*00{}=NYYYYYY
001C=NYYYYYY008
001CP1D=NYYYYYY008
001CP1DP1H=NYYYYYY008Minimum with 2--3!h
001CP1DP1HP1S=NYYYYYY0084+!h, 4!s, at most invitational
001CP1DP1HP1N=NYYYYYY0Sign off
001CP1DP1HP2C=NYYYYYY008Puppet to 2!d
001CP1DP1HP2CP2D=NYYYYYY008Forced
001CP1DP1HP2CP2DP2H=NYYYYYY008Mildly invitational with 5!h
001CP1DP1HP2CP2DP2S=NYYYYYY008Invitational, 5+!h and 4!s
001CP1DP1HP2CP2DP2N=NYYYYYY0Strongly invitational with 5!h
001CP1DP1HP2CP2DP3C=NYYYYYY008Invitational with 4!h and 5+ minor
001CP1DP1HP2CP2DP3D=NYYYYYY008Invitational with 4!h and 5+ minor
001CP1DP1HP2CP2DP3H=NYYYYYY0086!h, about 11--12 hcp
001CP1DP1HP2D=NYYYYYY008Artificial game force
001CP1DP1HP2H=NYYYYYY0086+!h, about 9--10 hcp
001CP1DP1S=NYYYYYY0085+!c, 4+!s, unlimited
001CP1DP1N=NYYYYYY017--19 bal, 2--3!h
001CP1DP2C=NYYYYYY0085+!c, unbal, 0--2!h, 0--3!s
001CP1DP2D=NYYYYYY008Reverse
001CP1DP2H=NYYYYYY008Minimum, 4!h
001CP1DP2S=NYYYYYY00816+ hcp, 5+!c and 4+!h
001CP1DP2SP3D=NYYYYYY008Retransfer
001CP1DP2SP3DP3H=NYYYYYY008
001CP1DP2SP3DP3HP3S=NYYYYYY008Cue bid, slam interest
001CP1DP2SP3DP3HP4C=NYYYYYY008Cue bid, slam interest
001CP1DP2SP3DP3HP4D=NYYYYYY008Cue bid, slam interest
001CP1DP2SP3DP3HP4H=NYYYYYY008To play
001CP1DP2SP3H=NYYYYYY008Invitational
001CP1DP2SP3S=NYYYYYY008Splinter
001CP1DP2SP4C=NYYYYYY008Splinter
001CP1DP2SP4D=NYYYYYY008Splinter
001CP1DP2SP4H=NYYYYYY008To play
001CP1DP2N=NYYYYYY016+ hcp, 6+!c. 18+ if 3!h
001CP1DP2NP3C=NYYYYYY008Suggestion to play
001CP1DP2NP3D=NYYYYYY008Relay
001CP1DP2NP3DP3H=NYYYYYY0083!h, 18+ hcp
001CP1DP2NP3H=NYYYYYY008Game forcing with 6+!h
001CP1DP3C=NYYYYYY00815--17 hcp, 6+!c and 3!h
001CP1DP3CP3D=NYYYYYY008Retransfer
001CP1DP3CP3H=NYYYYYY008Invitational
001CP1DP3D=NYYYYYY00817--19 bal, 4!h
001CP1DP3DP3H=NYYYYYY008To play
001CP1DP3H=NYYYYYY00813--15 hcp, good hand, 5+!c and 4!h
001CP1DP3HP3N=NYYYYYY0Asking for singleton
