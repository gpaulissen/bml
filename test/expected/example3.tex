\documentclass[a4paper]{article}
\usepackage[margin=1in]{geometry}
\usepackage[T1]{fontenc}
\usepackage[utf8]{inputenc}
\usepackage{newcent}
\usepackage{helvet}
\usepackage{graphicx}

\usepackage[pdftex, pdfborder={0 0 0}]{hyperref}
\frenchspacing

\usepackage{txfonts} % For \varheartsuit and \vardiamondsuit
\usepackage[usenames,dvipsnames]{color} % dvipsnames necessary to made PDFLaTeX work.
\usepackage{enumerate}
\usepackage{listliketab}
\usepackage{latexsym} % \Box
\usepackage{pbox} % \Box

% suits

%%% Colors

\newcommand{\BC}{\textcolor{OliveGreen}{$\clubsuit$}}
\newcommand{\BD}{\textcolor{RedOrange}{$\vardiamondsuit$}}
\newcommand{\BH}{\textcolor{Red}{$\varheartsuit${}}}
\newcommand{\BS}{\textcolor{Blue}{$\spadesuit${}}}

%suits for pdf-friendly titles
\newcommand{\pdfc}{\texorpdfstring{\BC{}}{C}}
\newcommand{\pdfd}{\texorpdfstring{\BD{}}{D}}
\newcommand{\pdfh}{\texorpdfstring{\BH{}}{H}}
\newcommand{\pdfs}{\texorpdfstring{\BS{}}{S}}

\newenvironment{bidtable}
{\begin{tabbing}

xxxxxx\=xxxxxx\=xxxxxx\=xxxxxx\=xxxxxx\=xxxxxx\=xxxxxx\=xxxxxx\=xxxxxx\=xxxxxx\=\kill}
{\end{tabbing} }%

\newenvironment{bidding}%
{\begin{tabbing}
xxxxxx\=xxxxxx\=xxxxxx\=xxxxxx \kill
}{\end{tabbing} }%end bidding


% writing hands
\newcommand{\cards}[1]{\textsf{#1}}
\newcommand{\spades}[1]{\BS\cards{#1}}
\newcommand{\hearts}[1]{\BH\cards{#1}}
\newcommand{\diamonds}[1]{\BD\cards{#1}}
\newcommand{\clubs}[1]{\BC\cards{#1}}
\newcommand{\void}{--}
\newcommand{\hand}[4]{\spades{#1}\ \hearts{#2}\ \diamonds{#3} \clubs{#4}}
\newcommand{\vhand}[4]{\spades{#1}\\\hearts{#2}\\\diamonds{#3}\\\clubs{#4}}

% The \Box should always appear the same distance from the left margin
\newcommand\onesuit[4]%
{%
  \begin{center}%
    \begin{tabular}{>{\hfill}p{3cm}cp{3cm}}
                & \cards{#2} \\%
      \cards{#1}& $\Box$    & \cards{#3} \\%
                & \cards{#4} %
    \end{tabular}
  \end{center}%
}

% A special command if the south hand is not shown to avoid whitespace
\newcommand\onesuitenw[3]%
{%
  \begin{center}%
    \begin{tabular}{>{\hfill}p{3cm}cp{3cm}}%
                & \cards{#2} \\%
      \cards{#1}& $\Box$    & \cards{#3}%
    \end{tabular}%
  \end{center}%
}

\newcommand\dealdiagram[5]%
{%
  \begin{center}%
    \begin{tabular}{>{\hfill}p{3cm}cp{3cm}}
      \pbox{20cm}{\small #5}& \pbox{20cm}{#2} \\%
      \pbox{20cm}{#1}& $\Box$    & \pbox{20cm}{#3} \\%
                & \pbox{20cm}{#4} %
    \end{tabular}
  \end{center}%
}

\newcommand\dealdiagramenw[4]%
{%
  \begin{center}%
    \begin{tabular}{>{\hfill}p{3cm}cp{3cm}}
      \pbox{20cm}{\small #4}& \pbox{20cm}{#2} \\%
      \pbox{20cm}{#1}& $\Box$    & \pbox{20cm}{#3} \\%
    \end{tabular}
  \end{center}%
}

\newcommand\dealdiagramew[2]%
{%
  \begin{center}%
    \begin{tabular}{>{\hfill}p{3cm}cp{3cm}}
      \pbox{20cm}{#1}& $\Box$    & \pbox{20cm}{#2} \\%
    \end{tabular}
  \end{center}%
}

\title{No title defined}
\author{No author defined}
\begin{document}
\maketitle
\tableofcontents

\begin{bidtable}
1\BC-1\BD\+\\
1\BH \> Minimum with 2--3\BH \+\\
1\BS \> 4+\BH , 4\BS , at most invitational\\
1NT \> Sign off\\
2\BC \> Puppet to 2\BD \+\\
2\BD \> Forced\+\\
2\BH \> Mildly invitational with 5\BH \\
2\BS \> Invitational, 5+\BH\ and 4\BS \\
2NT \> Strongly invitational with 5\BH \\
3m \> Invitational with 4\BH\ and 5+ minor\\
3\BH \> 6\BH , about 11--12 hcp\-\-\\
2\BD \> Artificial game force\\
2\BH \> 6+\BH , about 9--10 hcp\-\\
1\BS \> 5+\BC , 4+\BS , unlimited\\
1NT \> 17--19 bal, 2--3\BH \\
2\BC \> 5+\BC , unbal, 0--2\BH , 0--3\BS \\
2\BD \> Reverse\\
2\BH \> Minimum, 4\BH \\
2\BS \> 16+ hcp, 5+\BC\ and 4+\BH \+\\
3\BD \> Retransfer\+\\
3\BH\+\\
3\BS \> Cue bid, slam interest\\
4\BC\BD \> Cue bid, slam interest\\
4\BH \> To play\-\-\\
3\BH \> Invitational\\
3\BS \> Splinter\\
4\BC\BD \> Splinter\\
4\BH \> To play\-\\
2NT \> 16+ hcp, 6+\BC . 18+ if 3\BH \+\\
3\BC \> Suggestion to play\\
3\BD \> Relay\+\\
3\BH \> 3\BH , 18+ hcp\-\\
3\BH \> Game forcing with 6+\BH \-\\
3\BC \> 15--17 hcp, 6+\BC\ and 3\BH \+\\
3\BD \> Retransfer\\
3\BH \> Invitational\-\\
3\BD \> 17--19 bal, 4\BH \+\\
3\BH \> To play\-\\
3\BH \> 13--15 hcp, good hand, 5+\BC\ and 4\BH \+\\
3NT \> Asking for singleton\-\-
\end{bidtable}

\end{document}
